docker/Dockerfile

```diff
@@ -71,7 +71,7 @@ COPY --from=conda /opt/conda /opt/conda
 WORKDIR /workspace
 RUN git clone -b ${PT_BRANCH} ${PT_REPO} && \
     cd pytorch && git checkout ${PT_COMMIT} && git submodule sync && git submodule update --init --recursive && \
-    python setup.py develop && cd .. 
+    python setup.py develop && cd ..
 ARG TORCH_VISION_BRANCH=nightly
 ARG TORCH_VISION_COMMIT=${TORCH_VISION_BRANCH}
```

Comment:
Please update the defaut version based on the change in PR #14, and apply it to all related dockerfiles